persistence-jdbc/javadsl/src/test/scala/com/lightbend/lagom/javadsl/persistence/jdbc/JdbcPersistenceSpec.scala

```diff
@@ -29,17 +29,21 @@ abstract class JdbcPersistenceSpec(_system: ActorSystem) extends ActorSystemSpec
   import system.dispatcher
 
   protected lazy val slick = new SlickProvider(system)
-  protected lazy val offsetStore =
-    new JavadslJdbcOffsetStore(
-      slick,
-      system,
-      new OffsetTableConfiguration(
-        system.settings.config,
+
+  protected lazy val jdbcReadSide: JdbcReadSide = {
+    val offsetStore =
```

Comment:
`JpaReadSideImplSpec.scala` is failing to compile now, because it was also using the `offsetStore` value.